pages/runtime/runtime.md

```diff
@@ -1,21 +1,83 @@
 ---
-title: Runtime Environment
+title: Container Runtime
 ---
+# Container Runtime
+
+On resin.io devices all your application code runs a [ Docker container][container-link]. This means that whatever you define as `CMD` in your `Dockerfile` will be PID 1 of the process tree in your container. It also means that this PID 1 process needs to know how to properly process UNIX signals, reap orphan zombie processes [[1]][ref-one] and if it crashes your whole container crashes, losing logs and debug info.
```

Comment:
Can the `ref-one` be an inline reference? I don't have a problem with `[1]` in general, though we don't normally use this citation style in general, so it stands out. (not problem, just comment)